docs/extensibility.md

```diff
@@ -29,531 +28,630 @@ pub trait DiscoveryHandler {
+> **NOTE** the Akri SemVer (e.g. `0.0.41`) is reflected in `./version.txt` but the tags must be prefixed with `v` and postfixed with the architecture (e.g. `-amd64`)
+
+Check using `kubectl get pods` and look for a pod named `akri-agent-...` and another named `akri-controller...` and that they're both `RUNNING`.
 
-Assuming you have a Kubernetes cluster running (assuming amd64 for this sample), you can start Akri and apply your Nessie Configuration and watch as broker pods are created.
+Alternatively, you may:
 
-```sh
-# Add secret to give Kubernetes access to your container repo
-kubectl create secret docker-registry <SECRET NAME> --docker-server=ghcr.io  --docker-username=<GITHUB USERNAME> --docker-password=<GITHUB PAT>
-# Use Helm to install your nessie-enabled agent and controller
-helm install akri akri-<VERSION>.tgz \
-    --set imagePullSecrets[0].name="<SECRET NAME>" \
-    --set agent.image.repository="ghcr.io/<GITHUB USERNAME>/agent" \
-    --set agent.image.tag="extensibility-amd64" \
-    --set controller.image.repository="ghcr.io/<GITHUB USERNAME>/controller" \
-    --set controller.image.tag="extensibility-amd64"
-# Apply nessie Akri Configuration
-kubectl apply -f nessie.yaml
-# Watch as agent, controller, and nessie Pods start
-watch kubectl get pods -o wide
+```bash
+kubectl get pods --selector=name=akri-agent
+kubectl get pods --selector=app=akri-controller
 ```
 
+
+## Deploy Broker
+
+Once the HTTP broker has been created, the next question is how to deploy it.  For this, we need the Configuration we created earlier `samples/brokers/http/kubernetes/http.yaml`.  To deploy, use a simple `kubectl` command like this:
+```bash
+kubectl apply --filename=./samples/brokers/http/kubernetes/http.yaml
+```
+
+We can watch as the broker pods get deployed: 
+```bash
+microk8s watch kubectl get pods -o wide
+```
+
+
 ## Contributing your Protocol Implementation back to Akri
 Now that you have a working protocol implementation and broker, we'd love for you to contribute your code to Akri. The following steps will need to be completed to do so:
 1. Create an Issue with a feature request for this protocol.
 2. Create a proposal and put in PR for it to be added to the [proposals folder](./proposals).
 3. Implement your protocol and provide a full end to end sample.
 4. Create a pull request, updating the minor version of akri. See [contributing](./contributing.md#versioning) to learn more about our versioning strategy.
 
-For a protocol to be considered fully implemented the following must be included in the PR. Note how the Nessie protocol above only has completed the first 3 requirements. 
+For a protocol to be considered fully implemented the following must be included in the PR. Note that the HTTP protocol above has not completed all of the requirements. 

```

Comment:
Should we make this less strict? Thats a lot to ask someone to contribute. Maybe we say this is what we consider to be "fully implemented and worthy of being added to a release" but implementing just part is helpful?

Reply:
this seems like something we can do in another PR :).  the goal of this PR is to replace nessie with something more realistic, not change what this.